inventory_service/src/main.py

```diff
@@ -46,3 +46,4 @@ def delete_container(container_name):
     return items
 if __name__ == '__main__':
     app.run(debug=True,host = '127.0.0.1',port = 4000)
+    
```

Comment:
use delete method for delete functionality
![image](https://user-images.githubusercontent.com/70574489/217153460-5e59db18-4e38-4a06-ae69-c00eed959d85.png)